common/stages/Functions/answerfile

```diff
@@ -37,6 +37,14 @@ read_xml_tag()
     sed -ne 's/^.*<'"${TAG}"'\(\s\+[^>]*\)*\s*>\([^<]*\)<\/'"${TAG}"'>.*$/\2/p' <${FILE}
 }
 
+# This assumes the tag is on a single line in the file.
+sanitize_xml_tag()
+{
+    FILE="$1"
+    TAG="$2"
+    sed -i -e 's/\(^.*<'"${TAG}"'\(\s\+[^>]*\)*\s*>\)\([^<]*\)\(<\/'"${TAG}"'>.*$\)/\1SANITIZED\4/' ${FILE}
```

Comment:
Sweet! I suggest you amend the commit since there's already a lot of them in your PR! But a new commit works too if that's easier for you.